### ISSUE TITLE
H-Tags and Navbar fonts?

### ISSUE THREAD

**Issue by jsieler** [uikit/uikit#3044]
I always wondered, why do H-Tags (h1, uk-h1,..) and Navbar component do have separate font family? Basically font-family is already defined within html element, so all childs will have same font name.

So, what is the reason?

**Comment by Riddler84** [comment]
Basically the font is all the same as default. LESS and SASS files will use the global font-family. But you have the possibility to change the font-family for headings, navbar and pre-tags etc.

I think it's important to have the choice, because it can be often desired to use an other font for headlines as for normal text. For design reasons ;)

**Comment by zzseba78** [comment]
This is not an issue, it´s a feature, **and it´s great**! Combining fonts it´s a key feature for add meaning, hierarchy and constrast to designs. And there are several cases when you don´t want to apply to  the navbar items  the same typography of the paragraph.

**Comment by tyler-x** [comment]
Why does UIkit pre-define the font-family for headings and navbar? I think it's better to leave it blank or as default. So people who wants to use specific font can just define it globally.

People who want different font for headings and navs can simply define them by themselves.

**Comment by zzseba78** [comment]
@tyler-x Most of the time, designer are using a font pair, one font for body content and another for headings, special text, maybe buttons, or maybe navigation items.

It´s great to have the option, because, again, this is the most common practice, and if you want to use only one font family you just need to asign the same font family to the heading font.

**Comment by tyler-x** [comment]
I agree that a lot of projects require different font settings on body and headings, what I'm suggesting is UIkit to not pre-define the font family for these tags. As even if developers want to set different font for different tags, they will certainly not use the font UIkit pre-defined, so they will have to change it anyway.

So why wasting time pre-define something that most users will have to change anyway, whether they want to use different fonts for body and navs or not.

**Comment by zzseba78** [comment]
Maybe you are talking about css version.
In less for example, uikit defines the general font-family, and apply the same family ( less variable ) to headers and navbar, so in one place you can change all fonts, i don´t see the wasting time. And, if you need a diferent font for headers, specify the @base-heading-font-family.

https://github.com/uikit/uikit/blob/e02d1d6c59b2766d12541695e9ecaff8b4a5de5b/src/less/components/navbar.less#L45

https://github.com/uikit/uikit/blob/e02d1d6c59b2766d12541695e9ecaff8b4a5de5b/src/less/components/base.less#L40

**Comment by tyler-x** [comment]
Yes, you are right, I was referring to the CSS version. Most of my project are simple one page site, and I just quote the css/js from the CDN without hosting them. And most cases, I will have to override the font-family in my own style, and have to list all tags that have pre-defined fonts, which is somehow annoying.
